exampleCourse/questions/demoAutograderSquare/tests/initial_code.py

```diff
@@ -0,0 +1,5 @@
+import numpy as np
+
+# This default code is defined in "initial_code.py"!
```

Comment:
I'm not sure I understand the purpose of this comment.

Reply:
![image](https://user-images.githubusercontent.com/1790491/81483523-86f5c680-9204-11ea-9465-15af3063fd19.png)

Reply:
Got it!